util/commp.go

```diff
@@ -17,6 +17,7 @@ const DefaultMaxTraversalLinks = 2 << 29
 func CommP(ctx context.Context, bs bstore.Blockstore, root cid.Cid) (cid.Cid, abi.UnpaddedPieceSize, error) {
 	// do a CARv1 traversal with the DFS selector.
 	sc := car.NewSelectiveCar(ctx, bs, []car.Dag{{Root: root, Selector: shared.AllSelector()}}, car.MaxTraversalLinks(DefaultMaxTraversalLinks))
+	//sc := car.NewSelectiveCar(ctx, bs, []car.Dag{{Root: root, Selector: selectorparse.CommonSelector_ExploreAllRecursively}}, car.MaxTraversalLinks(DefaultMaxTraversalLinks))
```

Comment:
I am not sure if our code is working for CARv1 right now, so leaving this here, until we fix it. Actually `util.CommP` will probably go away altogether.